program/find-volume-of-a-cone/FindVolumeOfACone.js

```diff
@@ -0,0 +1,7 @@
+function getVolumeOfACone(radius, height) {
+    // Find volume using this formula:
+    // V = 1/3 × π × Radius^2 × Height
+    return (1/3) * Math.PI * (radius ** 2) * height;
+}
+
+findConeVolume(2, 3);
```

Comment:
Function name mismatch in the definition vs execution , update the function name 
`getVolumeOfACone(2, 3);`